lib/data/models/event_model.dart

```diff
@@ -7,8 +9,8 @@ class EventModel extends Equatable {
   final String status; // Offline/ Online Event
   final String certificate;
   final int date;
-  final String latitude; //used to add [Navigate to Venue]
-  final String longitude; //used to add [Navigate to Venue]
+  final double? latitude; //used to add [Navigate to Venue]
```

Comment:
can we store this as LatLng. as it is stored as LatLng under domain/entities/event.dart

Reply:
Since the data is coming from the model, we can't convert it there kadha anna, that's why we used the entity.